spec/amp-story-player.md

```diff
@@ -126,3 +130,179 @@ CSS variable with the URL pointing to the poster image of the story.
   </a>
 </amp-story-player>
 ```
+
+#### exit-control
+
+Set to `back-button` or `close-button`. The button will dispatch an `amp-story-player-back` or `amp-story-player-close` when clicked. This button will disappear when a page attachment is open and reappear when closed.
+
+## Programmatic Control
+
+Call the player's various methods to programmatically control the player.
+
+### Methods
+
+#### load
+
+Will initialize the player manually. This can be useful when the player is dynamically added such as when using frameworks like React.
+
+```
+function StoryPlayer({ url, title, poster, width = 360, height = 600 }) {
+
+       useEffect(() => {
+           const playerEl = document.body.querySelector('amp-story-player');
+           const player = new AmpStoryPlayer(window, playerEl); // Previously exposed global var
+           player.load();
+	}, []);
+
+	return (
+		<div>
+		  <amp-story-player style={{ width: `${width}px`, height: `${height}px`}}>
+			<a href={url}>
+			    {title}
+			</a>
+		  </amp-story-player>
+		</div>
+	);
+}
+```
+
+#### go
+
+- number: the story in the player to which you want to move, relative to the current story.
```

Comment:
Here and below: Let's add a "parameters" subheading before the parameters to provide a bit more clarity.